index.js

```diff
@@ -155,6 +155,18 @@ module.exports = postcss.plugin('postcss-mixins', function (opts) {
         }
     }
 
+    if ( !( 'contentKeyword' in opts ) ) {
+        opts.contentKeyword = 'mixin-content';
+    }
+
+    if ( !( 'defineKeyword' in opts ) ) {
+        opts.defineKeyword = 'define-mixin';
+    }
+
+    if ( !( 'usageKeyword' in opts ) ) {
+        opts.usageKeyword = 'mixin';
+    }
+
```

Comment:
Suggestion to use https://github.com/sindresorhus/object-assign to handle the defaults, instead:

```js
var assign = require('object-assign');
// ...
opts = assign({
  contentKeyword: 'mixin-content',
  defineKeyword: 'define-mixin',
  usageKeyword: 'mixin'
}, opts);
```